packages/fxa-settings/src/components/GetDataTrio/index.tsx

```diff
@@ -23,18 +23,32 @@ const DownloadContentTypeL10nMapping: Record<DownloadContentType, string> = {
   'Firefox account recovery key': 'get-data-trio-title-firefox-recovery-key',
 };
 
+export interface GetDataTrioGleanData {
+  id?:
```

Comment:
Should this be optional?

Reply:
Ah yes, thanks for catching. I made this optional before discussing with Ross that this will be our "required" property.